From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Mon, 14 Dec 2020 23:57:57 +0100
Subject: Add boards DTS to Makefile

> X-Git-Archeology: - Revision 2ccaee6343060b6614693fbcba36f8386457b850: https://github.com/armbian/build/commit/2ccaee6343060b6614693fbcba36f8386457b850
> X-Git-Archeology:   Date: Mon, 14 Dec 2020 23:57:57 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Enable station-m1 (roc-rk3328-pc) in rockchip64-dev too
> X-Git-Archeology:
> X-Git-Archeology: - Revision 1432a5cb96dc7dc290e48879de897af59bb954cb: https://github.com/armbian/build/commit/1432a5cb96dc7dc290e48879de897af59bb954cb
> X-Git-Archeology:   Date: Mon, 14 Dec 2020 23:57:57 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Consolidate all dts Makefile changes in a single patch in rockchip64-dev
> X-Git-Archeology:
> X-Git-Archeology: - Revision 19a963189510a541a0486933eb8eaa1d7bc7f695: https://github.com/armbian/build/commit/19a963189510a541a0486933eb8eaa1d7bc7f695
> X-Git-Archeology:   Date: Mon, 28 Dec 2020 22:44:02 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: - add Neo3 to DEV (#2504)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 18cf7aff702832cf672117f5dc1fe4ad37598837: https://github.com/armbian/build/commit/18cf7aff702832cf672117f5dc1fe4ad37598837
> X-Git-Archeology:   Date: Tue, 05 Jan 2021 23:35:03 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Consolidate all dts Makefile changes in a single patch in rockchip64-current (#2535)
> X-Git-Archeology:
> X-Git-Archeology: - Revision d5ab47bff7813b5e94ed239ec6007613740932ba: https://github.com/armbian/build/commit/d5ab47bff7813b5e94ed239ec6007613740932ba
> X-Git-Archeology:   Date: Sun, 10 Jan 2021 12:40:11 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Minor tweaks to NanoPi R4S configuration (#2543)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 804c57dd5925e67383a3d82c85f18868a6173d4b: https://github.com/armbian/build/commit/804c57dd5925e67383a3d82c85f18868a6173d4b
> X-Git-Archeology:   Date: Tue, 26 Jan 2021 21:22:37 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Enable RTC (hym8563) for Station P1 in mainline (renaming DT in process) (#2577)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 05a72b8954e932089d4867b55da0d353dc4ba1ac: https://github.com/armbian/build/commit/05a72b8954e932089d4867b55da0d353dc4ba1ac
> X-Git-Archeology:   Date: Mon, 12 Apr 2021 22:32:21 +0200
> X-Git-Archeology:   From: AmadeusGhost <42570690+AmadeusGhost@users.noreply.github.com>
> X-Git-Archeology:   Subject: rockchip: add Orange Pi R1 Plus support (#2755)
> X-Git-Archeology:
> X-Git-Archeology: - Revision e7377248b3cae186e24e2be781cd3365b43246f0: https://github.com/armbian/build/commit/e7377248b3cae186e24e2be781cd3365b43246f0
> X-Git-Archeology:   Date: Thu, 22 Jul 2021 00:15:54 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Second part of EDGE bumping to 5.13.y (#3045)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 744ea89a589d62cb6f409baab60fc6664520bc39: https://github.com/armbian/build/commit/744ea89a589d62cb6f409baab60fc6664520bc39
> X-Git-Archeology:   Date: Wed, 08 Sep 2021 17:51:34 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Bumping EDGE kernel to 5.14.y (#3125)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 687c3639183e23e06406739af1684fdeb3efa454: https://github.com/armbian/build/commit/687c3639183e23e06406739af1684fdeb3efa454
> X-Git-Archeology:   Date: Wed, 15 Sep 2021 22:20:35 +0200
> X-Git-Archeology:   From: Ruikai (Rick) Liu <rickliu2000@outlook.com>
> X-Git-Archeology:   Subject: Add NanoPi R2C support (#3138)
> X-Git-Archeology:
> X-Git-Archeology: - Revision dd51f9f2afcbc83a3e10b32eb6a5061d91d1558e: https://github.com/armbian/build/commit/dd51f9f2afcbc83a3e10b32eb6a5061d91d1558e
> X-Git-Archeology:   Date: Tue, 09 Nov 2021 18:06:34 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Bump imx6, xu4, rockchip64 and jetson-nano to 5.15 (#3238)
> X-Git-Archeology:
> X-Git-Archeology: - Revision ac8fc4385594d59257ee9dffd9efa85e3497fa7d: https://github.com/armbian/build/commit/ac8fc4385594d59257ee9dffd9efa85e3497fa7d
> X-Git-Archeology:   Date: Sat, 26 Feb 2022 07:46:44 +0100
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switch rockchip64 current to linux 5.15.y (#3489)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 6ff15fe54f0423267ef464927e3780bc47703eed: https://github.com/armbian/build/commit/6ff15fe54f0423267ef464927e3780bc47703eed
> X-Git-Archeology:   Date: Sat, 26 Feb 2022 15:03:33 +0100
> X-Git-Archeology:   From: schwar3kat <61094841+schwar3kat@users.noreply.github.com>
> X-Git-Archeology:   Subject: Add orangepi r1 plus lts (#3514)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 897674aa74bce0326ed7fe06f5336bf4709a8a1f: https://github.com/armbian/build/commit/897674aa74bce0326ed7fe06f5336bf4709a8a1f
> X-Git-Archeology:   Date: Tue, 03 May 2022 08:27:32 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Bump and freeze kernel at last known working versions (#3736)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 1f54bc33b8dda678de473d7d006a9a7f6ab0b86b: https://github.com/armbian/build/commit/1f54bc33b8dda678de473d7d006a9a7f6ab0b86b
> X-Git-Archeology:   Date: Wed, 11 May 2022 21:47:28 +0200
> X-Git-Archeology:   From: CXM <16154023+littlecxm@users.noreply.github.com>
> X-Git-Archeology:   Subject: Add clockworkpi-a06 support (#3768)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 597d2dac11f00d9070a4e49d6bad1b2244e36cb3: https://github.com/armbian/build/commit/597d2dac11f00d9070a4e49d6bad1b2244e36cb3
> X-Git-Archeology:   Date: Sat, 28 May 2022 07:56:22 +0200
> X-Git-Archeology:   From: Jianfeng Liu <liujianfeng1994@gmail.com>
> X-Git-Archeology:   Subject: update rockchip64-edge to 5.18 (#3814)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 388e9bbf5ebd83bad953ad0e0a7a0a41ba2a6ed8: https://github.com/armbian/build/commit/388e9bbf5ebd83bad953ad0e0a7a0a41ba2a6ed8
> X-Git-Archeology:   Date: Tue, 14 Jun 2022 12:11:33 +0200
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: Support for OrangePi4 LTS board (#3770)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2ca6a9381db4b875533926e0eae9d3d17f68ad06: https://github.com/armbian/build/commit/2ca6a9381db4b875533926e0eae9d3d17f68ad06
> X-Git-Archeology:   Date: Thu, 23 Jun 2022 08:30:54 +0200
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip64: add rk3318-box tvbox board patch and configurations (#3921)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 76192f1fa830a2c0fcf7a511bfbc119705b46a22: https://github.com/armbian/build/commit/76192f1fa830a2c0fcf7a511bfbc119705b46a22
> X-Git-Archeology:   Date: Wed, 06 Jul 2022 10:38:09 -0400
> X-Git-Archeology:   From: Tony <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: RK3328 cleanup (#3963)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 44ac745ae34bc3863e0a5a51d1f300968d426811: https://github.com/armbian/build/commit/44ac745ae34bc3863e0a5a51d1f300968d426811
> X-Git-Archeology:   Date: Sun, 10 Jul 2022 02:14:03 -0400
> X-Git-Archeology:   From: Tony <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: Rockchip64-current: fix nanoPi NEO 3 DTS makefile (#3979)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 8c6641e7b79f0d50acdc306d140e586a4e923cf0: https://github.com/armbian/build/commit/8c6641e7b79f0d50acdc306d140e586a4e923cf0
> X-Git-Archeology:   Date: Wed, 03 Aug 2022 22:22:55 +0200
> X-Git-Archeology:   From: Jianfeng Liu <liujianfeng1994@gmail.com>
> X-Git-Archeology:   Subject: update rockchip64 edge to 5.19 (#4039)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2f1140edb1078b4a6a3e73466df942aa36194c46: https://github.com/armbian/build/commit/2f1140edb1078b4a6a3e73466df942aa36194c46
> X-Git-Archeology:   Date: Tue, 20 Sep 2022 08:50:23 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Re-enable Neo3 in the EDGE kernel (#4193)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 6765f734cc4a22aeaa9f99a3ad28c8c322de26f6: https://github.com/armbian/build/commit/6765f734cc4a22aeaa9f99a3ad28c8c322de26f6
> X-Git-Archeology:   Date: Tue, 25 Oct 2022 11:26:51 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Bump rockchip64 edge to 6.0.y (#4337)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 92f1a22d76b987afa7ba555d5b509adc51d689e7: https://github.com/armbian/build/commit/92f1a22d76b987afa7ba555d5b509adc51d689e7
> X-Git-Archeology:   Date: Fri, 16 Dec 2022 13:38:13 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Re-add rockchip64 6.0 patches (#4575)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 588c2ec17e709dec19304fa50522459702ebfadd: https://github.com/armbian/build/commit/588c2ec17e709dec19304fa50522459702ebfadd
> X-Git-Archeology:   Date: Fri, 23 Dec 2022 21:57:53 +0100
> X-Git-Archeology:   From: brentr <brent@mbari.org>
> X-Git-Archeology:   Subject: Rockpis devtree mainlined (#4603)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 34ae84fac5d0b66a1ab2d1e51534b7beb13ef245: https://github.com/armbian/build/commit/34ae84fac5d0b66a1ab2d1e51534b7beb13ef245
> X-Git-Archeology:   Date: Fri, 05 May 2023 14:22:00 +0200
> X-Git-Archeology:   From: amazingfate <liujianfeng1994@gmail.com>
> X-Git-Archeology:   Subject: bump rockchip64 edge to v6.3
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/Makefile | 18 ++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index 8efdd3665331..fb635c76adb8 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -1,4 +1,22 @@
 # SPDX-License-Identifier: GPL-2.0
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-roc-pc.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-rock-pi-e.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3318-box.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-nanopi-neo3-rev02.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-nanopi-r2-rev00.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-nanopi-r2-rev06.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-nanopi-r2-rev20.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-orangepi-r1-plus.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-orangepi-r1-plus-lts.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-z28pro.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-kobol-helios64.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-m4v2.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-r4s.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-orangepi-4.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-orangepi-4-lts.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-roc-pc-plus.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-rock-pi-4.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-tinker-2.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += px30-evb.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += px30-engicam-px30-core-ctouch2.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += px30-engicam-px30-core-ctouch2-of10.dtb
-- 
Armbian

